projects/ui/src/components/Analytics/Bean/index.tsx

```diff
@@ -9,12 +9,12 @@ import Volume from '~/components/Analytics/Bean/Volume';
 import Liquidity from '~/components/Analytics/Bean/Liquidity';
 import Crosses from '~/components/Analytics/Bean/Crosses';
 import DeltaB from '~/components/Analytics/Bean/DeltaB';
-
-// const SLUGS = ['price', 'volume', 'liquidity', 'mktcap', 'supply', 'crosses'];
 import { FC } from '~/types';
 
+const SLUGS = ['price', 'volume', 'liquidity', 'mktcap', 'supply', 'crosses', 'DeltaB'];
```

Comment:
Instead of removing the space (which is important cosmetically in the UI), could you suggest a different approach that lets a dev add `label="Delta B"` but but keeps the slug url-safe?

I can think of two satisfactory approaches, let me know what you think of.